va/va.c

```diff
@@ -548,7 +548,6 @@ static VAStatus va_openDriver(VADisplay dpy, char *driver_name)
                     CHECK_VTABLE(vaStatus, ctx, EndPicture);
                     CHECK_VTABLE(vaStatus, ctx, SyncSurface);
                     CHECK_VTABLE(vaStatus, ctx, QuerySurfaceStatus);
-                    CHECK_VTABLE(vaStatus, ctx, PutSurface);
```

Comment:
Shouldn't they just return VA_STATUS_ERROR_UNIMPLEMENTED?

Reply:
if there are no implmented, it will set vaStatus = VA_STATUS_ERROR_UNKNOWN and return at https://github.com/intel/libva/blob/master/va/va.c#L572

Reply:
https://github.com/intel/libva/blob/master/va/va.c#L55

Reply:
Right. What I meant is, imo,  that the driver implementators should add a function that returns VA_STATUS_ERROR_UNIMPLEMENTED.

Meanwhile, returning VA_STATUS_ERROR_UNKNOWN might be misleading to the api user.

But anyway, I'm not opposing to this change. Just stating a point of view.

Reply:
@ceyusa  I filed another PR #404  to return  VA_STATUS_ERROR_UNIMPLEMENTED